MAVProxy/modules/mavproxy_chat/chat_openai.py

```diff
@@ -165,7 +185,10 @@ def send_to_assistant(self, text):
             reply_messages = self.client.beta.threads.messages.list(self.assistant_thread.id,
                                                                     order="asc",
                                                                     after=input_message.id)
-            if reply_messages is None:
+
+            if (self.latest_run.status == "cancelled"):
+                return "cancelled successfully"
+            elif reply_messages is None:
```

Comment:
Not a blocker but the "elif" could be just an "if" because of the return one line higher